docs/security-self-assessment.md

```diff
@@ -129,74 +124,85 @@ Every time a server endpoint is invoked, OpenFGA validates that:
 
 ## Project Compliance
 
-When using OpenFGA you need to store relationship tuples like `{user: user: alice, relation: can_view, object: document:readme }`. We recommend users to not to store  Personal Identifiable Information like email addresses in any of the relationship tuples. This will make it simpler to comply with GDPR and other privacy regulations.
+When utilizing OpenFGA, it's required to store relationship tuples like `{user: user: alice, relation: can_view, object: document:readme }`. We strongly advise users against storing Personal Identifiable Information (PII) such as email addresses in any of the relationship tuples. 
+
+This precaution is recommended to ensure compliance with GDPR and other privacy regulations.
+
+By refraining from including PII in relationship tuples, users can simplify their compliance efforts and mitigate potential privacy risks. This practice aligns with data protection principles and safeguards user privacy, contributing to a more secure and regulatory-compliant implementation of OpenFGA.
 
 ## Secure Development Practices
 
-The OpenFGA project follows established CNCF and OSS best practices for code development and delivery. OpenFGA [passes OpenSSF Best Practices](https://bestpractices.coreinfrastructure.org/projects/6374), has an [OpenSSF scorecard of](https://api.securityscorecards.dev/projects/github.com/openfga/openfga) 9 and a [CLO Monitor score of 99%](https://clomonitor.io/projects/openfga).
+The OpenFGA project include the test cases as per the CNCF standard. It passes the [OpenSSF](https://bestpractices.coreinfrastructure.org/projects/6374) Best Practices. SonarScan tells that the OpenFGA's code coverage is around 83% with A+ Go rating.
 
-OpenFGA's code coverage is 83%, and it has a [A+ Go rating](https://goreportcard.com/report/github.com/openfga/openfga).
+The OpenFGA project follows established CNCF and OSS best practices for code development and delivery. OpenFGA [passes OpenSSF Best Practices](https://bestpractices.coreinfrastructure.org/projects/6374), has an [OpenSSF scorecard of](https://api.securityscorecards.dev/projects/github.com/openfga/openfga) 9.1 and a [CLO Monitor score of 100%](https://clomonitor.io/projects/openfga).
+
+CodeCov [reports a code coverage of 82.10%](https://app.codecov.io/gh/openfga/openfga, and it has a [A+ Go rating](https://goreportcard.com/report/github.com/openfga/openfga).
 
 ### Ecosystem
 
 OpenFGA uses [Chainguard images](https://www.chainguard.dev/chainguard-images), it supports [OpenTelemetry](https://github.com/open-telemetry), and can be monitored with tools like [Grafana](https://grafana.com/), [Prometheus](https://prometheus.io/) and [Jaeger](https://www.jaegertracing.io/). 
 
+To monitor the logs and enabling the tracing mechanisms, OpenFGA can be integrated with [Jaeger](https://www.jaegertracing.io/). Jaeger adds the tracing headers to the logs, making easy to track the request flow. 
```

Comment:
```suggestion
Tracing can be enabled in OpenFGA and traces can be exported to [Jaeger](https://www.jaegertracing.io/).
```

The server adds the trace ID to the logs, not Jaeger. See https://github.com/openfga/openfga/issues/827